bundles/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/api/model/UniFiControllerRequest.java

```diff
@@ -136,6 +150,10 @@ private String getContent() throws UniFiException {
                 if (logger.isTraceEnabled()) {
                     logger.trace("<< {} {} \n{}", status, HttpStatus.getMessage(status), prettyPrintJson(content));
                 }
+
+                String csrfToken = response.getHeaders().get("X-CSRF-Token");
+                if (csrfToken != null && !csrfToken.isEmpty())
```

Comment:
`if(StringUtils.isNotEmpty(csrfToken))`

Reply:
Please don't use apache libraries anymore. We are trying to remove them.